Cargo.toml

```diff
@@ -91,10 +91,10 @@ num-traits = "0.2"
 oci-client = "0.15"
 openid = "0.18.3"
 openssl = "0.10"
-opentelemetry = "0.30"
-opentelemetry-otlp = "0.30"
-opentelemetry_sdk = "0.30"
-opentelemetry-instrumentation-actix-web = "0.22.0"
+opentelemetry = "0.31"
```

Comment:
:thinking: 

something is wrong
https://github.com/guacsec/trustify/pull/2016